spec/rubocop/target_ruby_spec.rb

```diff
@@ -308,33 +308,89 @@
             expect(target_ruby.version).to eq 2.7
           end
 
-          it 'sets target_ruby from required_ruby_version from inclusive range' do
+          it 'sets target_ruby from exacts string version' do
             content =
               <<-HEREDOC
                 Gem::Specification.new do |s|
                   s.name = 'test'
-                  s.required_ruby_version = '>= 3.0.0'
+                  s.required_ruby_version = '2.7.4'
```

Comment:
I wouldn't even test this. There's is no reason I can think of to specify such a requirement. We could even consider raising an offense in one of our cops for this, but I doubt it happens in practice.